### PR TITLE
javascript-modal | Rocky Truong

### PR DESCRIPTION
**Pull request by rocky-truong**
js modal demo

### CODE REVIEW

File: javascript-modal/main.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+var $openModalButton = document.querySelector('.open-modal-button');
+var $body = document.querySelector('.body');
+var $noButton = document.querySelector('.no-button');
+var $modal = document.querySelector('.modal');
+
+function modalHandler(event) {
```
Comment (codingcodymiller) [inline javascript-modal/main.js:6]: How about `openModal`?

File: javascript-modal/main.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  $modal.className = 'modal';
+  $body.className = 'body dimmed';
+  $openModalButton.className = 'open-modal-button dimmer';
+}
+
+$openModalButton.addEventListener('click', modalHandler);
+
+function noHandler(event) {
```
Comment (codingcodymiller) [inline javascript-modal/main.js:14]: How about `closeModal`?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work!! A couple notes for you:

**Comment by rocky-truong** [comment]
Uploading javascriptModal gif.mp4…

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**